src/Libraries/RevitNodes/Elements/Tag.cs

```diff
@@ -0,0 +1,269 @@
+        /// <param name="host"></param>
+        /// <param name="orientation"></param>
+        /// <param name="mode"></param>
+        /// <param name="addLeader"></param>
+        /// <param name="point"></param>
+        private void Init(
+            Autodesk.Revit.DB.View view,
+            Autodesk.Revit.DB.Element host,
+            Autodesk.Revit.DB.TagOrientation orientation,
+            Autodesk.Revit.DB.TagMode mode,
+            bool addLeader,
+            Autodesk.Revit.DB.XYZ point)
+        {
+
+            Autodesk.Revit.DB.Document document = DocumentManager.Instance.CurrentDBDocument;
+            TransactionManager.Instance.EnsureInTransaction(document);
+
+            var tagElem = ElementBinder.GetElementFromTrace<Autodesk.Revit.DB.IndependentTag>(document);
+
+            // create a new tag if the existing tag is null or its host view is not the selected one
+            // or the host is neither set a host or linked element.
+            if (tagElem == null || view.Id != tagElem.OwnerViewId ||
+                (tagElem.TaggedElementId.HostElementId != host.Id && tagElem.TaggedElementId.LinkedElementId != host.Id))
+            {
+                tagElem = document.Create.NewTag(view, host, addLeader, mode, orientation, point);
+            }
+            else
+            {
+                // apply properties
+                tagElem.TagOrientation = orientation;
+                tagElem.HasLeader = addLeader;
+                tagElem.TagHeadPosition = point;
+            }
+
+            double rotation = (orientation == TagOrientation.Horizontal) ? 0 : 90;
+            InternalSetType(tagElem.TagText, tagElem.TagHeadPosition, rotation);
+            InternalSetElement(tagElem);
+
+            TransactionManager.Instance.TransactionTaskDone();
+            ElementBinder.SetElementForTrace(this.InternalElement);
+        }
+
+        #endregion
+
+        #region Public static constructors
+
+        /// <summary>
+        /// Create a Revit Tag for a Revit Element
+        /// </summary>
+        /// <param name="view">View to Tag</param>
+        /// <param name="element">Element to tag</param>
+        /// <param name="horizontal">Horizontal alignment</param>
+        /// <param name="addLeader">Add a leader</param>
+        /// <param name="offset">Offset Vector or Tag Location</param>
+        /// <param name="isOffset">Specifies if the point is being used as an offset vector or if it specifies the tags location</param>
+        /// <param name="horizontalAlignment">Horizontal Alignment</param>
+        /// <param name="verticalAlignment">Vertical Alignment</param>
+        /// <returns></returns>
+        public static Tag ByElement(Revit.Elements.Views.View view, Element element, bool horizontal, bool addLeader, [DefaultArgument("null")]Autodesk.DesignScript.Geometry.Point offset, bool isOffset = true, string horizontalAlignment = "Center", string verticalAlignment = "Middle")
```

Comment:
I changed it to [DefaultArgument("Autodesk.DesignScript.Geometry.Vector.ByCoordinates(0,0,0)")]